packages/components/src/Spinner/Spinner.mdx

```diff
@@ -0,0 +1,53 @@
+---
+name: Spinner
+menu: Components
+route: /components/spinner
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Spinner } from ".";
+import { Text } from "../Text";
```

Comment:
you can import components in `mdx` files as `import { Text } from "@jobber/components/Text"`